lights-out.js

```diff
@@ -1,7 +1,22 @@
-class LightsOut1 {
+class LightsOut {
     constructor(gameId) {
         this.gameId = gameId;
         this.numRows = 4;
         this.numCols = 4;
+        this.randomize();
     }
-}
+
+    randomize()
+    {
+        for(let row = 0; row < this.numRows; row++)

```

Comment:
Please place opening bracket on same line of randomize(), like this:

```
randomize() { 
    ...
}
```

And, apply that same styling throughout your PR, for for loops, etc. 

Different projects have use different styles, primarily based on the subjective tastes of the leaders of projects.